exams/utils.py

```diff
@@ -66,9 +66,24 @@ def is_eligible_for_exam(mmtrack, course_run):
         course_run (courses.models.CourseRun): A CourseRun object.
 
     Returns:
-        bool: whether use is eligible or not
+        bool: whether user is eligible or not
     """
-    return course_run.has_exam and mmtrack.has_paid(course_run.edx_course_key)
+    return course_run.has_exam and has_paid(mmtrack, course_run.edx_course_key)
+
+
+def has_paid(mmtrack, edx_course_key):
```

Comment:
Could you prefix this with `_` so it's clear that this shouldn't be used elsewhere? Also, it should probably be named `_has_paid_for_exam` because this is specific to exams.